packages/material-ui/scripts/rollup.config.js

```diff
@@ -7,6 +7,23 @@ import nodeGlobals from 'rollup-plugin-node-globals';
 import { terser } from 'rollup-plugin-terser';
 import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
 
+// Resolve imports like:
+// import Portal from '@material-ui/unstyled/Portal';
```

Comment:
Shouldn't this be done by `babel-plugin-module-resolver`?

Reply:
It also seems to work with:

```diff
diff --git a/babel.config.js b/babel.config.js
index 658f4724f9..f2a14b0472 100644
--- a/babel.config.js
+++ b/babel.config.js
@@ -132,6 +132,19 @@ module.exports = function getBabelConfig(api) {
           ],
         ],
       },
+      rollup: {
+        plugins: [
+          [
+            'babel-plugin-module-resolver',
+            {
+              root: ['./'],
+              alias: {
+                '@material-ui/unstyled': path.resolve(__dirname, './packages/material-ui-unstyled/src'),
+              },
+            },
+          ],
+        ],
+      },
       benchmark: {
         plugins: [
           ...productionPlugins,
diff --git a/packages/material-ui/package.json b/packages/material-ui/package.json
index fad598de19..2b67f5fdcf 100644
--- a/packages/material-ui/package.json
+++ b/packages/material-ui/package.json
@@ -31,7 +31,7 @@
     "build:modern": "node ../../scripts/build modern",
     "build:node": "node ../../scripts/build node",
     "build:stable": "node ../../scripts/build stable",
-    "build:umd": "cross-env BABEL_ENV=stable rollup -c scripts/rollup.config.js",
+    "build:umd": "cross-env BABEL_ENV=rollup rollup -c scripts/rollup.config.js",
     "build:copy-files": "node ../../scripts/copy-files.js",
     "build:types": "node ../../scripts/buildTypes",
     "extract-error-codes": "cross-env MUI_EXTRACT_ERROR_CODES=true yarn build:modern",
diff --git a/packages/material-ui/scripts/rollup.config.js b/packages/material-ui/scripts/rollup.config.js
index 8112288071..19460964f1 100644
--- a/packages/material-ui/scripts/rollup.config.js
+++ b/packages/material-ui/scripts/rollup.config.js
@@ -94,7 +94,7 @@ export default [
     external: Object.keys(globals),
     plugins: [
       nodeResolve(nodeOptions),
-      nestedFolder,
+      // nestedFolder,
       babel(babelOptions),
       commonjs(commonjsOptions),
       nodeGlobals(), // Wait for https://github.com/cssinjs/jss/pull/893
```

Reply:
Fine for now. I'll improve it in a follow-up. Our babel config already has all the relevant information. We just need to reconcile it with the rollup externals.